charts/k8s-monitoring/templates/platform_specific/openshift/security-context-constraint.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and (eq .Values.cluster.platform "openshift") .Values.logs.enabled .Values.logs.pod_logs.enabled }}
```

Comment:
When comparing with the [openshift-logging scc](https://github.com/openshift/cluster-logging-operator/blob/master/internal/auth/securitycontextconstraint.go#L48), we're missing the `readOnlyFileSystem` option. We should be able to enable this

Reply:
I had to remove that because the Agent fails when it tries to write to `/tmp/agent`.
I could add that back in and mount an `emptyDir` volume for temp storage

Reply:
Alright, I figured out how the openshift log collector does it.

They store log position information in `/var/lib/vector` and as this is a mounted volume (not the root volume) they can set keep the root volume to read only.

We could try it with `/var/lib/grafana-agent`